Encountify.Android/CustomMapRenderer.cs

```diff
@@ -14,7 +14,7 @@
 using Encountify.ViewModels;
```

Comment:
If CustomMapRenderer is in Encountify.Android, does it mean that map will not be rendered in iOS?

Reply:
By my guess most likely not. Haven't tested it out. It's something that could be done later on but is not a major priority now.